minigame/pkmnquiz_blue.asm

```diff
@@ -524,16 +529,17 @@ ENDM
 	lang D, next "3. Verschwinde"
 
 	; French
-	lang F, db   "Que disait le"
-	lang F, next "SPECTRE non iden"
-	lang F, next "tifié de la TOUR"
-	lang F, next "#MON en combat?"
+	lang F, db   "Que dit un SPECTRE"
```

Comment:
I was going to point out that present tense looks wrong, since we're talking about past events (and the original text uses past tense).

I checked for consistency with the other questions, and... turns out it's a bit all over the place :upside_down_face: 

IMO we should stick to past tense, for the above reasons, but that's extra characters so it may be difficult.

What do you think?